index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HANGMAN</title>
+    <meta name="description" content="hangman">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Coiny&amp;subset=latin-ext" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class = 'container'>
```

Comment:
please remove those spaces around `=`. It looks ugly and it's not consistent with `haead` tag. In general, in html convention is to write html attributes without spaces around `=`.